src/metaImage.cxx

```diff
@@ -767,8 +767,8 @@ ElementByteOrderSwap(METAIO_STL::streamoff _quantity)
       }
     case 2:
       {
-      int i;
-      for(i=0; i<quantity*m_ElementNumberOfChannels; i++)
+      for(size_t i=0;
+        i<static_cast<size_t>(quantity*m_ElementNumberOfChannels); i++)
```

Comment:
Instead, would it make sense to change the type of `m_ElementNumberOfChannels` from `int` to `size_t` ?